### PR TITLE
Travis and Appveyor config files

### PR DESCRIPTION
**Pull request by jkeenan**
Update MANIFEST and SKIP.

### CODE REVIEW

**Comment by jkeenan** [comment]
This p.r. supersedes the part of https://github.com/Perl-Toolchain-Gang/ExtUtils-Install/pull/18 supplying CI configuration files.  I am separating this part out so that each p.r. is more tightly focused.

Currently, the master branch of this repo has no CI files.  While the ones I am supplying may not be the optimal files, they work.  I request that they be applied to master so that subsequent pull requests get the benefit of CI.  If you want to perfect these CI files, feel free to do so -- but let's not let the perfect be the enemy of the good.

Thank you very much.
Jim Keenan

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  - "5.28-shrplib"
+  - "5.26-shrplib"
+  - "5.24-shrplib"
+  - "5.22-shrplib"
+  - "5.20-shrplib"
+  - "5.18-shrplib"
+notifications:
+  email:
+    recipients:
+      - jkeenan@cpan.org
```
Comment (karenetheridge) [inline .travis.yml]: I don't think this should remain in the master version.  If the notification section is left out, by default it is always the commit owner who is notified, which should be sufficient in most cases.

**Comment by Leont** [comment]
This removes the META files from the dist, that is not a good idea

File: .travis.yml (hunk 1 above)
Comment (jkeenan) [reply .travis.yml]: At this point, feel free to make any appropriate revisions in either `.travis.yml` or `.appveyor.yml`. As long as there are such files (which I provided in response to mohawk's suggestion), I'm happy.

**Comment by mohawk2** [comment]
@Leont No, it does not.

File: .travis.yml (hunk 1 above)
Comment (karenetheridge) [reply .travis.yml]: Well if you're happy getting notified for every travis failure going forward, have fun I guess :)

**Pull request merged**